packages/product-components/src/components/SelectAssetModal/SendTokenTabs.tsx

```diff
@@ -0,0 +1,57 @@
+import { Icon, Row, useElevation } from '@trezor/components';
+import { borders, Elevation, mapElevationToBorder, spacings, spacingsPx } from '@trezor/theme';
+import { TokenFilterCategory } from './SelectAssetModal';
+import styled from 'styled-components';
+import { CheckableTag } from './CheckableTag';
+
+interface NetworkTabsWrapperProps {
+    $elevation: Elevation;
+}
+
+const NetworkTabsWrapper = styled.div<NetworkTabsWrapperProps>`
+    margin-left: -${spacingsPx.md};
+    width: calc(100% + ${spacings.md * 2}px);
```

Comment:
Seems more elegant to me than `calc`.
And works the same - the fine divider line spans the whole width :heavy_check_mark: 
```suggestion
    margin: 0 -${spacingsPx.md};
```